docs/cloud-provider-specifics.md

```diff
@@ -181,75 +185,108 @@ Now that you have your Google Cloud credentials stored in a Secret, you need to
 
 ### Azure
 
+#### Kubernetes cluster prerequisites
+
+Ensure that the VMs for your agent pool allow Managed Disks. If I/O performance is critical,
+consider using Premium Managed Disks, as these are SSD backed.
+
 #### Service principal creation
 To integrate Heptio Ark with Azure, you should follow the instructions below to create an Ark-specific [service principal][17].
 
 1. If you do not have the `az` Azure CLI 2.0 locally installed, follow the [user guide][18] to set it up. Once done, run:
 
-    ```
+    ```bash
     az login
     ```
 
 2. There are seven environment variables that need to be set for Heptio Ark to work properly. The following steps detail how to acquire these, in the process of setting up the necessary RBAC.
 
-3. List your account:
+3. Obtain your Azure Account Subscription ID and Tenant ID:
 
+    ```bash
+    AZURE_SUBSCRIPTION_ID=`az account list --query '[?isDefault].id' -o tsv`
+    AZURE_TENANT_ID=`az account list --query '[?isDefault].tenantId' -o tsv`
     ```
-    az account list
-    ```
-    Save the relevant response values into environment variables: `id` corresponds to `$AZURE_SUBSCRIPTION_ID` and `tenantId` corresponds to `$AZURE_TENANT_ID`.
 
-4. Assuming that you already have a running Kubernetes cluster on Azure, you should have a corresponding resource group as well. List your current groups to find it:
+4. Set the name of the Resource Group that contains your Kubernetes cluster.
 
+    ```bash
+    # Change "Kubernetes" as needed
+    AZURE_RESOURCE_GROUP=Kubernetes
     ```
-    az group list
-    ```
-     Get your cluster's group `name` from the response, and use it to set `$AZURE_RESOURCE_GROUP`. (Also note the `location`--this is later used in the Azure-specific portion of the Ark Config).
 
-5. Create a service principal with the "Contributor" role:
+    If you are unsure of the Resource Group name, run the following command to get a list that you can select from. Then set the `AZURE_RESOURCE_GROUP` environment variable to the appropriate value.
 
+    ```bash
+    az group list --query '[].{ ResourceGroup: name, Location:location }'
     ```
-    az ad sp create-for-rbac --role="Contributor" --name="heptio-ark"
-    ```
-    From the response, save `appId` into `$AZURE_CLIENT_ID` and `password` into `$AZURE_CLIENT_SECRET`.
 
-6. Login into the `heptio-ark` service principal account:
+    Get your cluster's group `name` from the response, and use it to set `$AZURE_RESOURCE_GROUP`. (Also note the `location`--this is later used in the Azure-specific portion of the Ark Config).
```

Comment:
Get your cluster's Resource Group name from the `ResourceGroup` value in the response, and use it to set `$AZURE_RESOURCE_GROUP`. (Also note the `Location` value in the response - this is later used in the Azure-specific portion of the Ark Config).